helix-lsp/Cargo.toml

```diff
@@ -31,3 +31,4 @@ parking_lot.workspace = true
 arc-swap = "1"
 slotmap.workspace = true
 thiserror.workspace = true
+simd-json = "0.14.3"
```

Comment:
Looks like this is behind a bit, there's a v0.15 that transitively depends on hashbrown v0.15+ rather than v0.14. We have a few other dependencies still on v0.14 but those should be updated eventually (mainly looks like we'd be waiting on gix)